docs/reference/generated/form.json

```diff
@@ -3,19 +3,19 @@
   "description": "A native form element with consolidated error handling.\nRenders a `<form>` element.",
   "props": {
     "errors": {
-      "type": "Record<string, string | string[]>",
+      "type": "{}",
```

Comment:
Where does this type difference come from?

Reply:
It was a bug in the API extractor